packages/cli/src/api/catalog.ts

```diff
@@ -280,10 +280,6 @@ export class Catalog {
   ) {
     const catalog = catalogs[locale] || {}
 
-    if (!catalog.hasOwnProperty(key)) {
-      console.error(`Message with key ${key} is missing in locale ${locale}`)
```

Comment:
This might be useful. Maybe it would be better to change it to the `console.warn()` and make it non-blocking? Or print it only in `--verbose` mode

Reply:
It's a duplicating logic, which is left here by historical reasons. There is a "allowEmpty" option which is aimed for exactly that purpose. When "allowEmpty: false" (wich is usually in production) the CLI will exit with error and message. 

When `allowEmpty = false && verbose = true` it will also print all messages which is empty.  
Check the tests i wrote for more info. 

This piece of code was not covered by tests, and probably it was an unreachable path before i implemeted "*.pot" files as a source for keys in this PR https://github.com/lingui/js-lingui/pull/1296